### PR TITLE
boards: arm: arty: note which boards can use the pre-built bitstream

### PR DESCRIPTION
**Pull request by henrikbrixandersen**
Add a note on which Arty boards can be programmed with the pre-built FPGA bitstream.

Signed-off-by: Henrik Brix Andersen <henrik@brixandersen.dk>

### CODE REVIEW

**Review by nashif — APPROVED** [review]

File: boards/arm/arty/doc/index.rst [hunk 1]
```diff
@@ -133,6 +133,12 @@ or:
         pld load 0 m3_for_arty_a7_reference.bit;\
         shutdown"
 
+.. note::
+
+   The pre-built FPGA bitstream only works for Arty boards equipped with an
+   Artix-35T FPGA. For other Arty variants (e.g. the Arty A7-100) the bitstream
```
Comment (pabigot) [inline boards/arm/arty/doc/index.rst:139]: s/A7-100/A7-100T/

**Review by pabigot — CHANGES_REQUESTED** [review]
board identifier slightly incorrect.

File: boards/arm/arty/doc/index.rst (hunk 1 above)
Comment (henrikbrixandersen) [reply boards/arm/arty/doc/index.rst:139]: Not according to https://store.digilentinc.com/arty ?

Comment (pabigot) [reply boards/arm/arty/doc/index.rst:139]: Hmn, they're not entirely consistent.  From https://store.digilentinc.com/arty-a7-artix-7-fpga-development-board-for-makers-and-hobbyists/:

> Arty A7 comes in two FPGA variants: Arty A7-35T features Xilinx XC7A35TICSG324-1L. Arty A7-100T features the larger Xilinx XC7A100TCSG324-1. Arty A7 variants have the features listed below:

I got thrown by having `Artix-35T` in the same sentence.  If you're consistent in using `A7-35` everywhere, then `A7-100` is...consistent.

Removed nak, and thanks for doing this.

**Review by pabigot — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**